app/build.gradle.kts

```diff
@@ -117,15 +117,16 @@ android {
 dependencies {
 	implementation("androidx.core:core-ktx:1.13.1")
 	implementation("androidx.activity:activity-compose:1.9.0")
-	implementation("androidx.compose.material3:material3:1.3.0-beta01")
-	implementation("androidx.compose.material:material-icons-extended-android:1.6.7")
-	implementation("androidx.compose.ui:ui-tooling-preview:1.6.7")
-	debugImplementation("androidx.compose.ui:ui-tooling:1.6.7")
+	implementation("androidx.compose.material3:material3:1.3.0-beta04")
```

Comment:
Build error: This breaks `PreferenceDialogs.rememberTimePickerState()` because `TimePickerState.Saver()` was removed in `beta02`. I suggest reverting to `beta01` and I'll handle the migration.